portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,138 @@
+.header{
+    padding: 10px;
+    text-align: center;
+    background: #de5246;
+    color: white;
+    font-size: 30px;
+}
+
+.menu {
+  background-color: #4285F4;
+  overflow: hidden;
+}
+
+.menu a {
+  float:left;
+  width: 23.2%;
+  color: #ffffff;
+  text-align: center;
+  padding: 14px 16px;
+  font-size: 24px;
+  text-decoration: none;
+}
+
+.menu a:hover {
+  background-color: #1a1a1a;
+}
+
+.menu a.active {
+  background-color: #0f9d58;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 70%;
+
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+img.helmet {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  height:15%;
+  width:15%;
+}
+
+.row {
+  height:100%;
+  width:100%;
+  display: flex;
+}
+
+.column {
+  flex:33.33%;
+  padding:5px;
+}
+
+.twoRow {
+  height:100%;
+  width:100%;
+  display: flex;
+}
+
+.twoColumn {
+  flex:50%;
+  padding:5px;
+}
+
+.logo {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  top: 150px;
+}
+
+.contact {
+    position: fixed;
+    bottom: 5px;
+}
+
+.fa {
+    padding: 20px;
+    font-size: 30px;
+    width: 30px;
+    text-align: center;
+    text-decoration: none;
+    margin: 5px 2px;
+    border-radius: 50%;
+}
+
+.fa:hover {
+    opacity: 0.7;
+}
+
+.fa-linkedin {
+    background: #007bb5;
+    color: white;
+}
+
+.fa-github {
+    background: #2c4762;
+    color: white;
+}
```

Comment:
2 space indention to follow style guide and maintain consistency. Try this tool to help you format:  https://github.com/stekolla/sps-clang-format